pymc/distributions/distribution.py

```diff
@@ -544,6 +544,10 @@ def __new__(
         rv_out._repr_latex_ = types.MethodType(
             functools.partial(str_for_dist, formatting="latex"), rv_out
         )
+
+        # https://docs.marimo.io/guides/integrating_with_marimo/displaying_objects/#option-1-implement-a-_display_-method
+        rv_out._display_ = types.MethodType(functools.partial(safe_display, model=model), rv_out)
```

Comment:
I think it's a good time as any to remove pretty print from the variables and leave it to the model graphviz only.

There was an issue/ PR to remove those others for ipython. This is never going to be exhaustive because there are many other ways of creating variables for models

Reply:
So does that mean, removing other pretty print methods as well? I'm not following your comment fully.